config/routes.rb

```diff
@@ -5,11 +5,26 @@
   # You can have the root of your site routed with "root"
   root 'task#index'
 
-  get '/show/:selected_id' => 'task#show'
+  get '/show/:selected_id'      => 'task#show'
 
```

Comment:
Most of your routes are well set up - I think after the additional work you've done with the RESTful routes now you can see how you'd want to change these to follow the RESTful standards a bit more